net-proxy/trojan/trojan-1.6.0.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="An unidentifiable mechanism that helps you bypass GFW."
+HOMEPAGE="https://trojan-gfw.github.io/trojan/"
+SRC_URI="https://github.com/trojan-gfw/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="alpha amd64 x86 arm arm64 hppa mips ppc ppc64 s390 sh sparc"
```

Comment:
barely you tested it on them all, also do not commit straight to stable. So i guess your proper keywords are ~amd64 and ~x86

Reply:
It 's stable now,and I've tested on `amd64 x86 arm arm64 `, I'll delete other